front/app/components/FeatureFlag/index.tsx

```diff
@@ -12,7 +12,7 @@ import {
 import { isNilOrError } from 'utils/helperUtils';
 
 interface Props {
```

Comment:
Made it easier to use the component. Otherwise you'd have to check name before rendering FeatureFlag, now it happens in the component (as was the original intent I believe, if I look at the code at line 68.